src/pages/ManageApps.vue

```diff
@@ -279,7 +288,12 @@ function ellipsisAppName(appName: string) {
                 />
               </button>
               <img :src="getImageUrl(app.id)" class="app-logo" />
-              <VStack gap="0.5rem" style="max-width: 100%">
+              <VStack
+                gap="0.5rem"
+                style="max-width: 100%"
+                @mouseenter="setHover(app.id, true)"
+                @mouseleave="setHover(app.id, false)"
```

Comment:
Need to use better strategy for this, can be done directly with tailwind/css instead of js

Sample code
```vue
<VStack class="group/app-card">
  <img src="someimageurl" class="sm:invisible sm:group-hover/visible" />
  </VStack>
```

Can try something like this. Should work. We don't need to maintain a js state for that in the memory